File: trains-gdx-desktop/src/main/java/com/badlogic/test/java/TestDesktop.java
package com.badlogic.trainsgdx.java;

import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration;

import com.badlogic.trainsgdx.core.Test;

public class TestDesktop {
	public static void main (String[] args) {
		LwjglApplicationConfiguration config = new LwjglApplicationConfiguration();
		new LwjglApplication(new Test(), config);
	}
}

File: trains-gdx-desktop/src/main/kotlin/com/badlogic/test/java/TestDesktop.kt
package com.badlogic.test.java

import com.badlogic.gdx.backends.lwjgl.LwjglApplication
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration
import com.badlogic.trainsgdx.core.Test

object TestDesktop {
    @JvmStatic
    fun main(args: Array<String>) {
        val config = LwjglApplicationConfiguration()
        LwjglApplication(Test(), config)
    }
}